### ISSUE TITLE
Unable to create comet logger when using pytorch lightning cli.

### ISSUE THREAD

**Issue by george-gca** [PyTorchLightning/pytorch-lightning#12734]
## 🐛 Bug

Unable to create comet logger when using pytorch lightning cli.

### To Reproduce
https://colab.research.google.com/drive/1cvEyYHceKjunKpcGY39oFrinWnIVydJV?usp=sharing

### Expected behavior
Run model.

### Environment

* CUDA:
	- GPU:
		- Tesla T4
	- available:         True
	- version:           11.1
* Packages:
	- numpy:             1.21.5
	- pyTorch_debug:     False
	- pyTorch_version:   1.10.0+cu111
	- pytorch-lightning: 1.6.0
	- tqdm:              4.63.0
* System:
	- OS:                Linux
	- architecture:
		- 64bit
		- 
	- processor:         x86_64
	- python:            3.7.13
	- version:           1 SMP Tue Dec 7 09:58:10 PST 2021

### Additional context

Error message:
```
Epoch 1: 100% 32/32 [00:00<00:00, 300.70it/s, loss=-15.4, v_num=ff79]Traceback (most recent call last):
  File "main.py", line 48, in <module>
    cli = LightningCLI(BoringModel, LitDataset, save_config_callback=None)
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/utilities/cli.py", line 564, in __init__
    self._run_subcommand(self.subcommand)
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/utilities/cli.py", line 835, in _run_subcommand
    fn(**fn_kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/trainer/trainer.py", line 772, in fit
    self._fit_impl, model, train_dataloaders, val_dataloaders, datamodule, ckpt_path
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/trainer/trainer.py", line 724, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/trainer/trainer.py", line 812, in _fit_impl
    results = self._run(model, ckpt_path=self.ckpt_path)
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/trainer/trainer.py", line 1237, in _run
    results = self._run_stage()
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/trainer/trainer.py", line 1324, in _run_stage
    return self._run_train()
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/trainer/trainer.py", line 1354, in _run_train
    self.fit_loop.run()
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/loops/base.py", line 204, in run
    self.advance(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/loops/fit_loop.py", line 269, in advance
    self._outputs = self.epoch_loop.run(self._data_fetcher)
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/loops/base.py", line 204, in run
    self.advance(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/loops/epoch/training_epoch_loop.py", line 246, in advance
    self.trainer._logger_connector.update_train_step_metrics()
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py", line 202, in update_train_step_metrics
    self.log_metrics(self.metrics["log"])
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py", line 130, in log_metrics
    logger.log_metrics(metrics=scalar_metrics, step=step)
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/utilities/rank_zero.py", line 32, in wrapped_fn
    return fn(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/loggers/comet.py", line 252, in log_metrics
    self.experiment.log_metrics(metrics_without_epoch, step=step, epoch=epoch)
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/loggers/base.py", line 41, in experiment
    return get_experiment() or DummyExperiment()
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/utilities/rank_zero.py", line 32, in wrapped_fn
    return fn(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/loggers/base.py", line 39, in get_experiment
    return fn(self)
  File "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/loggers/comet.py", line 223, in experiment
    offline_directory=self.save_dir, project_name=self._project_name, **self._kwargs
TypeError: __init__() got an unexpected keyword argument 'agg_key_funcs'
```
For some reason, `self._kwargs` there has `{'agg_key_funcs': None, 'agg_default_func': None}`.

cc @awaelchli @edward-io @borda @ananthsub @rohitgr7 @kamil-kaczmarek @Raalsky @Blaizzy

**Comment by tshu-w** [comment]
see https://github.com/PyTorchLightning/pytorch-lightning/issues/12529 and https://github.com/PyTorchLightning/pytorch-lightning/issues/12714

**Comment by awaelchli** [comment]
This was fixed and included in the 1.6.1 release. Could you try upgrading lightning? `pip install --upgrade pytorch-lightning`

**Comment by george-gca** [comment]
@awaelchli not the same bug, but my colab link to reproduce the bug is now throwing another error.
